timebinmgr.c

```diff
@@ -35,7 +35,7 @@ setup_sync_points(double TimeIC, double no_snapshot_until_time)
     SyncPoints[1].a = All.TimeMax;
     SyncPoints[1].loga = log(All.TimeMax);
     SyncPoints[1].write_snapshot = 1;
-    SyncPoints[1].write_fof = 0;
+    SyncPoints[1].write_fof = All.SnapshotWithFOF ? 1 : 0;
```

Comment:
Is there a problem with the old one? If TimeMax was not in output list it won't write FOF, but a snapshot is dumpped just in case.

Reply:
Oh - I didn't realise that. I generally don't put TimeMax in the output list because at one point there was a bug that it would write two snapshots. If that is the case I can drop this commit.